src/transactionsFactories/tokenManagementTransactionsFactory.ts

```diff
@@ -120,7 +122,8 @@ export class TokenManagementTransactionsFactory {
             options.canAddSpecialRoles ? this.trueAsHex : this.falseAsHex,
         ];
 
-        return new DraftTransactionBuilder({
+        
```

Comment:
Remove one empty line unless the formatter added it, then you can ignore the comments. Applies to all methods.